CHANGES/435.feature

```diff
@@ -0,0 +1,4 @@
+Allow to use mod operator (`%`) for updating query string:
+
+    url = URL("http://example.com")
+    assert url % {"a": "1"} == URL("http://example.com?a=1")
```

Comment:
```suggestion
Allow using `mod` operator (`%`) for updating query string (an alias for `update_query()` method).
```

This file will be rendered as a list bullet, not a paragraph. I understand your desire to provide an usage example but sorry, it will not be rendered properly in this context.